src/Components/Artwork/GridItem.tsx

```diff
@@ -8,36 +9,36 @@ import { StoredImage } from "Utils/localImagesHelpers"
 import { cropped, resized } from "Utils/resized"
 import { userIsTeam } from "Utils/user"
 import { GridItem_artwork$data } from "__generated__/GridItem_artwork.graphql"
-import { MagnifyImage } from "Components/MagnifyImage"
 import Badge from "./Badge"
 import Metadata from "./Metadata"
 import { useHoverMetadata } from "./useHoverMetadata"
-import { useFeatureFlag } from "System/useFeatureFlag"
 
 interface ArtworkGridItemProps extends React.HTMLAttributes<HTMLDivElement> {
   artwork: GridItem_artwork$data
   contextModule?: AuthContextModule
   disableRouterLinking?: boolean
   hideSaleInfo?: boolean
-  isMyCollectionArtwork?: boolean
   lazyLoad?: boolean
   localHeroImage?: StoredImage | null
   onClick?: () => void
+  showHighDemandIcon?: boolean
   showHoverDetails?: boolean
   showSaveButton?: boolean
+  to?: string | null
 }
 
 export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = ({
   artwork,
   contextModule,
   disableRouterLinking,
   hideSaleInfo,
-  isMyCollectionArtwork = false,
   lazyLoad = true,
   localHeroImage,
   onClick,
+  showHighDemandIcon = false,
```

Comment:
Instead of a `isMyCollectionArtwork` prop I've added the `showHighDemandIcon` prop.